server/datastore/mysql/software.go

```diff
@@ -3211,7 +3213,7 @@ last_vpp_install AS (
 	return software, metaData, nil
 }
 
-func (ds *Datastore) SetHostSoftwareInstallResult(ctx context.Context, result *fleet.HostSoftwareInstallResultPayload) error {
+func (ds *Datastore) SetHostSoftwareInstallResult(ctx context.Context, result *fleet.HostSoftwareInstallResultPayload) (wasCanceled bool, err error) {
```

Comment:
Something felt a little odd to me to have `false` represent the "happy" states (pending or completed but not canceled) path of an activity, but I don't know if there's a great alternative. I fumbled around a bit searching for another name for the variable and the best I came up with were "applicable" or "included" but they don't feel like a perfect fit. No need to change anything at this point, just sharing what I came up with in case it is at all helpful.

Reply:
Yeah, I'm with you, it's much nicer how I could add it to the scripts, but unfortunately without making huge changes for software install we just have this call.